infra/src/main/kotlin/fi/fta/geoviite/infra/common/AlignmentDescription.kt

```diff
@@ -0,0 +1,43 @@
+package fi.fta.geoviite.infra.common
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonCreator.Mode.DELEGATING
+import com.fasterxml.jackson.annotation.JsonValue
+import fi.fta.geoviite.infra.util.StringSanitizer
+
+const val ALLOWED_DESCRIPTION_CHARACTERS = "A-ZÄÖÅa-zäöå0-9 _\\\\\\-–—+().,'\"/<>:;!?&"
+
+data class LocationTrackDescriptionBase @JsonCreator(mode = DELEGATING) constructor(private val value: String) :
```

Comment:
🤔 Kuinkas tällä mahtaa olla tuo Base-suffiksi?

Reply:
Tämä kuvaa sijaintiraiteen kuvauksen perusosaa, joka on meillä englanninnettu termille `DescriptionBase`. Meillä on vielä erillinen, sijaintiraiteen yhdistettyä kuvausta kuvaava `LocationTrackDescription`

Reply:
Äsh, aivan. Oletin hieman että toi oli jäänyt epähuomiossa jonkun perintähierarkian vuoksi. Base on nimittäin aika yleisesti käytetty sellaisissa "pohjaluokissa", joista muita luokkia peritään